mbtools/models.py

```diff
@@ -120,39 +141,74 @@ def delete_empty_categories(self):
                 elem.getparent().remove(elem)
 
 
+class MoodleLessonPage:
+    def __init__(self, tree, location):
+        self.tree = tree
+        self.location = location + f' (page: {self.name()})'
+        self.id = int(self.tree.attrib['id'])
+
+    def name(self):
+        return self.tree.xpath("title")[0].text
+
+    def next(self):
+        return int(self.tree.xpath("nextpageid")[0].text)
+
+    def prev(self):
+        return int(self.tree.xpath("prevpageid")[0].text)
+
+    def answers(self):
+        answer_objs = []
+        for answer in self.tree.xpath("answers/answer"):
+            answer_objs.append(
+                MoodleLessonAnswer(answer, self.location)
+            )
+        return answer_objs
+
+    def html_element(self):
+        return MoodleHtmlElement(self.tree.xpath("contents")[0], self.location)
+
+    def html_elements(self):
+        elems = []
+        elems.append(self.html_element())
+        for answer in self.answers():
+            elems.append(answer.html_element())
+        return elems
+
+
+class MoodleLessonAnswer:
+    def __init__(self, tree, location):
+        self.tree = tree
+        self.location = location + f' (answer id: {self.tree.attrib["id"]})'
```

Comment:
I see, because these are both technically "edges" to `MoodleHtmlElements`... In that case I suppose this is not the end of the world. I don't know if there are more moodle structures that are allowed in `LessonPages`, but we can try to be consistent if we come across them and use this method.